src/main/java/com/cst438/controller/AssignmentController.java

```diff
@@ -163,11 +194,12 @@ public List<AssignmentStudentDTO> getStudentAssignments(
             @RequestParam("year") int year,
```

Comment:
This function needs to implement the findBy() method stated in the comment and return the DTO list.